test/test_gpu_indexing_ops.cpp

```diff
@@ -44,8 +44,7 @@ TEST_F(NVFuserTest, FusionSelectOpPointwise_CUDA) {
   FusionExecutorCache executor_cache(std::move(fusion_ptr));
   auto cg_outputs = executor_cache.runFusionWithInputs({t0, idx});
 
-  testValidate(
-      &fusion, cg_outputs, {t0, idx}, {t1, t2, t3}, __LINE__, __FILE__);
+  testValidate(&fusion, cg_outputs, {t0, idx}, __LINE__, __FILE__);
```

Comment:
Could you remove the computation of `t1`, `t2`, and `t3` above? Similar for other changes in this PR. Wondering why clang-tidy not complaining unused variables.

Reply:
Right, I missed this. I'll do this in another PR when I add evaluate method for `ReductionOp`.